.travis.yml

```diff
@@ -30,11 +30,11 @@ matrix:
     - compiler: ": ARM"
       env: HOST=arm-linux-gnueabihf PACKAGES="g++-arm-linux-gnueabihf" DEP_OPTS="NO_QT=1" GOAL="install" BITCOIN_CONFIG="--enable-glibc-back-compat --enable-reduce-exports"
     - compiler: ": Win32"
-      env: HOST=i686-w64-mingw32 PACKAGES="nsis gcc-mingw-w64-i686 g++-mingw-w64-i686 binutils-mingw-w64-i686 mingw-w64-dev wine bc" RUN_TESTS=true GOAL="deploy" BITCOIN_CONFIG="--enable-gui --enable-reduce-exports" MAKEJOBS="-j2"
+      env: HOST=i686-w64-mingw32 PACKAGES="nsis gcc-mingw-w64-i686 g++-mingw-w64-i686 binutils-mingw-w64-i686 mingw-w64-dev wine bc" RUN_TESTS=false GOAL="deploy" BITCOIN_CONFIG="--enable-gui --enable-reduce-exports" MAKEJOBS="-j2"
```

Comment:
It's half-correct. If `RUN_TESTS` is set to true, not only the unit tests also the rpc test get tested. Because this PR introduces Win32 compatibility of the rpc tests, and they don't run properly on travis/Win32, they need to be disabled. But *only* the RPC tests (so maybe a new flag in `travis.yml` for RUN_RPC_TESTS).

But much better it would be, if the RPC test would succeed on Win32/Travis. I'm pretty convinced that there must be a bug somewhere if the RPC tests do not succeed on Win32.